docs/src/main/sphinx/connector/faker.md

```diff
@@ -0,0 +1,130 @@
+# Faker connector
+
+The Faker connector generates realistically looking random data matching a
+specific schema.
+
+## Configuration
+
+To configure the Faker connector, create a catalog properties file
+`etc/catalog/example.properties` with the following contents:
+
+```text
+connector.name=faker
+```
+
+## Examples
+
+Faker generates data when reading from a table created in a catalog using this
```

Comment:
I think we have to break the whole flow here up into separate steps .. here is what I think you want to say

* There is an existing table somewhere with a specific setup for columns and data types production.public.customer
* you can create a table with faker that can act as data generator using that format in a catalog called "example" (see configuration) that uses the faker connector .. if you want you can also change the name to generator or so .. but NOT faker
* related question .. do you have to create the default schema in it?
* can you create other schema in it?
* we need to document the CREATE TABLE LIKE syntax with this PR and link to it

Reply:
That's correct. The first paragraph is supposed to explain why and how to work with existing schemas in other catalogs.

I replaced the `example` catalog with `generator`, much better and avoids repeating `example`.

It already includes the `default` schema, and is ready to use out of the box. You can create additional schemas to match other catalogs and/or to use different defaults for null probability and default limit (same as the scale factor in tpch).

WDYT link to `CREATE TABLE LIKE`? It's documented in [CREATE TABLE](https://trino.io/docs/current/sql/create-table.html), I don't think we need a separate page for it, but maybe a linkable section. LMK if that's what you mean. BTW `CREATE TABLE AS SELECT * FROM other_catalog.table WITH NO DATA` would also work, but it's more verbose.

Reply:
Yeah .. sorry .. I was looking at old docs .. we should probably link to that section to help users

Reply:
I'll mark this as resolved for now, but I didn't add any extra links. If you think it's needed, please comment on specific parts again.